deepdoc/server/deepdoc_server.py

```
@@ -47,43 +57,99 @@ def parse_args():
+    if dla_adapter is None:
+        return JSONResponse(status_code=503, content={"error": "DLA endpoint disabled"})
+
+    data = await file.read()
+    result = dla_adapter(data)
+    return {"bboxes": result}
+
+
+@app.post("/predict/ocr")
+async def predict_ocr(
+    operator: str = Form(...),
+    file: UploadFile = File(...),
```

Ruff (0.16.1)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)